west.yml

```diff
@@ -36,7 +36,7 @@ manifest:
     - name: zephyr
       repo-path: fw-nrfconnect-zephyr
       west-commands: scripts/west-commands.yml
-      revision: ec52cc75e99ca715f8d6d5c0aed906a79b1e2879
+      revision: 56f6a33a71c46ec5b2e1beb6076edf9521014ad3
```

Comment:
GitHub says that this is a diff, but it is in sync with ncs/master AFAICS

Reply:
I doubt it ...

Reply:
https://github.com/NordicPlayground/fw-nrfconnect-nrf/blob/master/west.yml#L39

Reply:
I was wrong ...